modules/apps/asset/asset-list-service/src/main/java/com/liferay/asset/list/internal/upgrade/v1_4_0/util/AssetListEntryUsageTable.java

```diff
@@ -94,9 +94,11 @@
 	public static final String[] TABLE_SQL_ADD_INDEXES = {
 		"create index IX_76643456 on AssetListEntryUsage (assetListEntryId, classNameId, ctCollectionId)",
 		"create index IX_6E1C4974 on AssetListEntryUsage (assetListEntryId, ctCollectionId)",
-		"create unique index IX_78341F6B on AssetListEntryUsage (classNameId, classPK, portletId[$COLUMN_LENGTH:200$], ctCollectionId)",
```

Comment:
@YurenaC why has this file changed? it is from version 1.4.0 and we should not have changed it.

Reply:
I'm not sure why that line, and two more, where changed,... maybe every time we run an _ant all_?
Any clues on that @4lejandrito ?

Reply:
Change automatically generated